srtcore/buffer.cpp

```diff
@@ -2121,6 +2124,25 @@ int CRcvBuffer::extractData(char* data, int len, int p, int q, bool passack)
     return len - rs;
 }
 
+string CRcvBuffer::describeState(size_t first_n_pkts) const
```

Comment:
As it's a debug-only function, it better have a name starting with `debug`. Kinda, `debugTimeState`?

Reply:
Applied suggested name.